Gemfile.lock

```diff
@@ -186,6 +195,17 @@ GEM
     tzinfo (1.2.7)
       thread_safe (~> 0.1)
     unicode-display_width (1.7.0)
+    unicorn (4.9.0)
```

Comment:
Is it going to mess w/ things for the optional group to be in the gems from the lock?